js_modules/dagit/packages/core/src/assets/LaunchAssetChoosePartitionsDialog.tsx

```diff
@@ -246,6 +248,14 @@ const LaunchAssetChoosePartitionsDialogBody: React.FC<Props> = ({
       return;
     }
 
+    if (!canLaunchPipelineExecution.enabled) {
+      // Should never happen, this is essentially an assertion failure
+      showCustomAlert({
+        title: 'Unable to launch as single run',
```

Comment:
this title is weird, i see its copied from somewhere else - i wont' sweat it if it's an assert